frameworks/FSharp/giraffe/src/App/Program.fs

```diff
@@ -0,0 +1,36 @@
+module App.App
+
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Giraffe
+
+[<CLIMutable>]
+type JsonMessage = { message : string }
+
+let jsonutf8 (data:obj) : HttpHandler =

```

Comment:
Should this be `data: JsonMessage`?

Also is `type JsonMessage` the equivalent of a C# struct type? (rather than a class type) 

The generic Utf8Json Serialize overload for will work better for the struct type (don't change it for the regular one as Json.Net would box it to obj anyway...)

Reply:
It should be a class
Will add struct for utf8 case + make it generic